airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/util/ConnectorExceptionUtil.java

```diff
@@ -0,0 +1,55 @@
+package io.airbyte.integrations.util;
+
+import io.airbyte.commons.exceptions.ConfigErrorException;
+import io.airbyte.commons.exceptions.ConnectionErrorException;
+import io.airbyte.integrations.base.errors.messages.ErrorMessage;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.function.Predicate;
+
```

Comment:
nit : Can we add a comment to this class?

Something like: 
/*
Utility class defining methods for handling configuration exceptions in connectors.
*/